lib/LaTeXML/Util/Image.pm

```diff
@@ -311,7 +311,7 @@ sub image_graphicx_complex {
           else                     { $a1 = $w * $a2 / $h; } }
         ($w, $h) = (ceil($a1 * $dppt), ceil($a2 * $dppt)); } }
     my $X = 4;                 # Expansion factor
-    my ($dx, $dy) = (int($X * 72 * $w / $w0), int($X * 72 * $h / $h0));
+    my ($dx, $dy) = (int($X * 3 * 72 * $w / $w0), int($X * 3 * 72 * $h / $h0));
```

Comment:
I simply multiplied the density by 3 on each component, the rest is whitespace changes from latexmllint.